monitor/plugins/web_app/plugin.py

```diff
@@ -16,22 +16,22 @@
 import paramiko
 import time
 
-from monitor.monasca.manager import MonascaMonitor
+from monitor.utils.monasca.client import MonascaClient
 from monitor.plugins.base import Plugin
 
 
 class WebAppMonitor(Plugin):
 
-    def __init__(self, app_id, info_plugin, collect_period, keypair, retries=60):
-        Plugin.__init__(self, app_id, info_plugin, collect_period, retries=retries)
+    def __init__(self, app_id, info_plugin, keypair, retries=60):
+        Plugin.__init__(self, app_id, info_plugin, collect_period=5, retries=retries)
         self.app_id = app_id
         self.host_ip = info_plugin['host_ip']
         self.keypair_path = keypair
         self.host_username = info_plugin['host_username']
         self.log_path = info_plugin['log_path']
         self.dimensions = {'app_id': self.app_id, 'host': self.host_ip}
         self.last_checked = ''
-        self.monasca = MonascaMonitor()
+        self.monasca = MonascaClient()
```

Comment:
MonascaClient can be confuse here, because the official module is called monasca client.

Reply:
MonascaConnector look better to you?

Reply:
Much better!